cypress/integration/v2/features/global-settings/global-colors.spec.js

```diff
@@ -0,0 +1,437 @@
+/**
+ * External dependencies
+ */
+import { registerBlockSnapshots } from '~gutenberg-e2e/plugins'
+import { blocks } from '~stackable-e2e/config'
+
+const colors = [
+	{
+		name: 'Custom Color Red',
+		color: '#ff0000',
+	},
+	{
+		name: 'Aqua',
+		color: '#3fcee8',
+	},
+	{
+		name: 'Yellow Sun',
+		color: '#f2e374',
+	},
+	{
+		name: 'Stackable Pink',
+		color: '#f00069',
+	},
+]
+
+const blocksWithTitle = [
+	'ugb/accordion',
+	'ugb/heading',
+	'ugb/text',
+	'ugb/icon',
+	'ugb/feature-grid',
+	'ugb/image-box',
+	'ugb/feature',
+	'ugb/cta',
+	'ugb/card',
+	'ugb/header',
+	'ugb/count-up',
+	'ugb/pricing-box',
+	'ugb/notification',
+	'ugb/number-box',
+	'ugb/expand',
+]
+
+const blocksWithSeparator = [
+	'ugb/columns',
+	'ugb/container',
+	'ugb/icon-list',
+	'ugb/video-popup',
+	'ugb/testimonial',
+	'ugb/team-member',
+	'ugb/button',
+	'ugb/blockquote',
+]
+
+describe( 'Global Settings', () => {
+	it( 'should adjust global colors and assert the color picker in blocks', () => {
+		cy.setupWP()
+		cy.newPage()
+
+		// Add Global colors
+		cy.addBlock( 'core/paragraph' )
+		colors.forEach( val => {
+			cy.addEditGlobalColor( {
+				name: val.name,
+				color: val.color,
+			} )
+		} )
+		cy.adjust( 'Use only Stackable colors', true )
```

Comment:
Add blog/posts global colors assertion before calling the `registerBlockSnapshots` function